bus/routes.go

```diff
@@ -107,7 +107,7 @@ func (b *Bus) accountsFundHandler(jc jape.Context) {
 		}
 	} else {
 		// latest revision
-		rev, err := b.rhp3Client.Revision(jc.Request.Context(), req.ContractID, cm.HostKey, cm.SiamuxAddr)
+		rev, err := b.rhp3Client.Revision(jc.Request.Context(), req.ContractID, cm.HostKey, host.Info().SiamuxAddr)
```

Comment:
Instead of `host.Info()` which is just a helper to convert a host into the `HostInfo` type, I'd directly go for `host.Settings.SiamuxAddr()`.